### ISSUE TITLE
Error extracting large file into pfs

### ISSUE THREAD

**Issue by derekchiang** [pachyderm/pachyderm#282]
The transform:
```
tar xvjf wikipedia_50GB.tar.bz2 -C /pfs/out
```

Log from job:

```
wikipedia_50GB/
wikipedia_50GB/file64
wikipedia_50GB/file221
tar: wikipedia_50GB/file64: Cannot close: Input/output error
wikipedia_50GB/file80
2016/04/12 22:45:10 transport: http2Client.notifyError got notified that the client transport was broken EOF.
wikipedia_50GB/file219
tar: wikipedia_50GB/file80: Cannot close: Input/output error
tar: wikipedia_50GB: Cannot mkdir: Input/output error
wikipedia_50GB/file21
tar: wikipedia_50GB/file219: Cannot open: No such file or directory
tar: wikipedia_50GB: Cannot mkdir: Input/output error
wikipedia_50GB/file213
tar: wikipedia_50GB/file21: Cannot open: No such file or directory
tar: wikipedia_50GB: Cannot mkdir: Input/output error
tar: wikipedia_50GB/file213: Cannot open: No such file or directory
wikipedia_50GB/file93
tar: wikipedia_50GB: Cannot mkdir: Input/output error
wikipedia_50GB/file77
tar: wikipedia_50GB/file93: Cannot open: No such file or directory
tar: wikipedia_50GB: Cannot mkdir: Input/output error
```

**Comment by derekchiang** [comment]
Repo size increased to around 1GB before the errors occurred, at which point repo size becomes zero (as returned by `pachctl list-repo`).

**Comment by derekchiang** [comment]
Actually, it's unclear if this error was caused by me running some destructive commands accidentally.  Will try to reproduce this and close it if I can't.

**Comment by derekchiang** [comment]
Seems reproduce-able:

```
wikipedia_50GB/
wikipedia_50GB/file64
wikipedia_50GB/file221
wikipedia_50GB/file80
wikipedia_50GB/file219
tar: wikipedia_50GB/file80: Cannot close: Input/output error
wikipedia_50GB/file21
wikipedia_50GB/file213
tar: wikipedia_50GB/file21: Cannot close: Input/output error
wikipedia_50GB/file93
```

This time repo size gets stuck at 3GB.

**Comment by derekchiang** [comment]
Had to do with one of the bugs that affected #270.  Fixed in #292.
